test/javascripts/acceptance/chat-test.js

```diff
@@ -486,6 +486,17 @@ acceptance("Discourse Chat - without unread", function (needs) {
     assert.equal(query(".tc-composer-input").value.trim(), "");
   });
 
+  test("Pressing escape cancels editing", async function (assert) {
+    await visit("/chat/channel/9/Site");
+    const chatMessage = query(".chat-message");
+    await click(chatMessage.querySelector(".edit-btn"));
+    assert.ok(query(".tc-composer .tc-composer-message-details"));
+    await triggerKeyEvent(".tc-composer", "keydown", 27); // 27 is escape
+
+    // tc-composer-message-details will be gone as no message is being edited
+    assert.notOk(query(".tc-composer .tc-composer-message-details"));
```

Comment:
Please use `exists()` helper instead of `quert()` 🙂

Reply:
fine